OpenUpSA/wazimap-ng-ui
# Filtering choropleth on multiple columns

## Description
`and` filtering the choropleth on multiple columns

## Related Issue
https://github.com/OpenUpSA/wazimap-ng-ui/issues/348

## How to test it locally
* Select a subindicator from the data mapper to display choropleth
* Using the mapchip in the bottom of the page, add multiple filters
* Confirm that the choropleth values are filtered correctly


## Screenshots
![image](https://user-images.githubusercontent.com/53019884/120191769-02b51d80-c223-11eb-88f0-d66c6e46d52b.png)


## Changelog

### Added
* `src/js/elements/dropdown_menu.js` to manage dropdown elements

### Updated
* `src/css/wazimap-ng-v1.webflow.css` and `index.html` to apply the design changes
* `src/js/elements/mapchip.js` to manage the dropdowns, filters, buttons on the mapchip
* `src/js/profile/subindicator_filter.js` to filter on multiple columns

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
